examples/js/web-worker/README.md

```diff
@@ -0,0 +1,38 @@
+# Web-Worker
+
+This is an example project showing how to use `@bugsnag/web-worker` with a web worker.
+
+The project contains 2 pages, both of which spawn their own dedicated web worker and service worker.
+
+`web-app-reports-unhandled` has BugSnag instantiated in the workers AND the web app.
```

Comment:
I think with these lines explain a little why we would have this difference (e.g. metadata & breadcrumbs and stability scores).  If you don't know what I mean yell, although hopefully Tom's docs PR will illuminate.